### DISCUSSION TITLE
Any way to make Rose Pine (light theme) not repeat folder colors, like Latte?

### DISCUSSION THREAD

**Discussion by Chaesur** [AnubisNekhet/AnuPpuccin#226] [Snippets]
I tried adding a new option/variant in the Extended Themes snippet by duplicating the "Atom" light variation and then replacing its tri-color codes with the ones originally in Latte (minus the ones that are duplicates). But it turned out pretty poorly. 

It's probably obvious but I have no experience with CSS. Any ideas on how I can create this Rose Pine variant "Rose Pine Bloom"?

A) Should I have done Color Override on the duplicate colors with Style Settings, and then used the Custom Rainbow Colors snippet to set the order I wanted? 

B) Or should I have created the slight-variant duplicate in the main plug-ins CSS, and changed the duplicate color codes there?

I really like the Rose Pine theme, but the repeating folder colors are like a rock in my brain's shoe that I can't get past.

**Comment by AnubisNekhet** [comment]
Hi there!
AnuPpuccin follows the Catppuccin color scheme, which results in palettes having fewer accent colors getting duplicate accents. You can use the [Custom Rainbow Folder Colors snippet](https://github.com/AnubisNekhet/AnuPpuccin/blob/main/snippets/custom-rainbow-colors.css) to pick and choose what accent colors you want in your rainbow folder pattern.
